PXDConverter/MarshalService.cs

```diff
@@ -32,32 +39,58 @@ public static void InitializeDll()
             }
         }
 
-        public static void CloseDll()
+        public void CloseDll()
         {
             Console.WriteLine("Closing .dll file...");
-            
+
             try
             {
                 Close();
             }
             catch
             {
                 Console.WriteLine("Couldn't close the pxd32d5_d4.dll!");
-                Environment.Exit(-3);
+                Environment.Exit(-5);
             }
         }
 
-        public static void ConvertWavToRawDataBuffer(string wavPath, string bufferTmpPath)
+        public void Decompress(string pxdPath, int leftOffset, int leftSize, int rightOffset, int rightSize, int sampleRate, string outputPath)
```

Comment:
Are parameters: int leftOffset, int leftSize, int rightOffset, int rightSize, int sampleRate, string outputPath
actually needed? It seems that you don't use them in this method scope

Reply:
it was from the interface, will remove them